src/common/ManagedTelemetry/Telemetry/Events/EventBase.cs

```diff
@@ -12,5 +13,13 @@ namespace Microsoft.PowerToys.Telemetry.Events
     public class EventBase
     {
         public bool UTCReplace_AppSessionGuid => true;
+
+        public string Version
+        {
+            get
+            {
+                return CommonManaged.GetProductVersion();

```

Comment:
we should probably cache that, no need to call interop on every telemetry event as product version does not change

```
private static string version;
...
if(string.IsNullOrEmpty(version))
{
version = CommonManaged.GetProductVersion();
} 
return version;
```